Cargo.toml

```diff
@@ -27,8 +27,10 @@ tonic = { version = "0.8.0", optional = true }
 prost = { version = "0.11.0", optional = true }
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"], optional = true }
 uuid = { version = "1.1.2", features = ["serde", "v4"], optional = true }
```

Comment:
We need to bump uuid because virtee/sev requires `^1.2` (https://github.com/virtee/sev/blob/v1.1.0/Cargo.toml#L40).

Reply:
Shouldn't virtee/sev pull in its own dependencies? Or is there some incompatibility between the versions?

Reply:
It builds for me. That's kind of odd. No problem to update it.